test/controllers/InvalidDataControllerSpec.scala

```diff
@@ -79,7 +79,7 @@ class InvalidDataControllerSpec extends ControllerBaseSpec {
       val document = Jsoup.parse(contentAsString(result))
 
       status(result) shouldBe 200
-      document.toString should include("Sorry, there is a problem with the service")
+      document.toString should include("Sorry, the service is unavailable")
```

Comment:
Could you test against the actual messages in this file, please? Means we don't have to go back and change these tests whenever content changes.